time_tracker_api/api.py

```diff
@@ -0,0 +1,12 @@
+
+from flask_restplus import Api
+
+api = Api(version='1.0.1', title="TimeTracker API",
+          description="API for the TimeTracker project")
+
+# APIs
+from time_tracker_api.projects import projects_api
```

Comment:
Shouldn't we put all the imports at the top? 

On the other hand, the import `from time_tracker_api.projects import projects_api` from what I understand, you import the file `projects_api` and use it to reference functions inside of it. But It took me some minutes to figure it out. I think it would more easy to grasp something like `from time_tracker_api.projects.projects_api import ns as projects_ns`. But feel free to argument why your approach.

Reply:
The thing is that when I see the variable projects_ns it looks like an alias but it is not clear where it comes from, also it makes that import line bigger. When I see projects_api.ns, I understand that it comes from projects_api and that the variable it references is `ns`, there is no possible abstract interpretation.

Reply:
I did the import in the same part where I added that namespace because it seemed a good idea to have them together in case I wanted to remove or comment the inclusion of a particular namespace.